presto-parquet/src/main/java/io/prestosql/parquet/reader/TimestampMicrosColumnReader.java

```diff
@@ -40,9 +44,15 @@ protected void readValue(BlockBuilder blockBuilder, Type type)
         if (definitionLevel == columnDescriptor.getMaxDefinitionLevel()) {
             long epochMicros = valuesReader.readLong();
             // TODO: specialize the class at creation time
-            if (type.equals(TIMESTAMP_MICROS)) {
+            if (type.equals(TIMESTAMP_MILLIS)) {
```

Comment:
Should we do the same for TimestampWithTimeZone?

Reply:
added